integration/testdata/fixtures/db/opensuse.yaml

```diff
@@ -9,4 +9,15 @@
       pairs:
         - key: "openSUSE-SU-2020:0062-1"
           value:
-            FixedVersion: 1.1.0i-lp151.8.6.1
\ No newline at end of file
+            FixedVersion: 1.1.0i-lp151.8.6.1
+- bucket: "openSUSE Tumbleweed"
+  pairs:
+    - bucket: libopenssl3
+      pairs:
+        - key: "openSUSE-SU-2024:13065-1"
+          value:
+            FixedVersion: 3.1.1-3.1
```

Comment:
Let's update `FixedVersion` to show vulnerability for `opensuse-tumbleweed.tar.gz`
```suggestion
            FixedVersion: 3.1.5-9.1 # changed for test
```

After these changes you will need to update the gold file.